### PR TITLE
Add suggestions for possible PSR-7 implementations to composer.json

### PR DESCRIPTION
**Pull request by michaelbiberich**
References: #2653, specifically [this comment](https://github.com/slimphp/Slim/issues/2653#issuecomment-487377115).

### CODE REVIEW

**Review by l0gicgate — CHANGES_REQUESTED** [review]
I think that we should suggest Slim PSR-7 only and provide a link to the docs instead for more repos maybe?

```
"suggest": {
    "slim/psr7": "Slim PSR-7 implementation. See http://dev.slimframework.com/docs/v4/start/installation.html for more information.",
}
```

**Comment by michaelbiberich** [comment]
The requested changes have been addressed in https://github.com/slimphp/Slim/pull/2674/commits/aa6f672bc8fe5c73f0b3a2220022082c9fefc0b1.

File: composer.json [hunk 1]
```diff
@@ -76,5 +76,8 @@
         "phpunit": "php vendor/bin/phpunit",
         "phpcs": "php vendor/bin/phpcs",
         "phpstan": "php -d memory_limit=-1 vendor/bin/phpstan analyse Slim"
+    },
+    "suggest": {
+        "slim/psr7": "Slim PSR-7 implementation. See http://dev.slimframework.com/docs/v4/start/installation.html for more information."
```
Comment (l0gicgate) [inline composer.json]: Let's change that url to `http://www.slimframework.com/docs/v4/start/installation.html`

I don't want to have to PR this again on next release. Sorry 😂

Comment (michaelbiberich) [reply composer.json]: No problem, should be addressed in https://github.com/slimphp/Slim/pull/2674/commits/249a5852060edde0c861f68d860b726d112d3ade.

**Review by l0gicgate — APPROVED** [review]

**Comment by l0gicgate** [comment]
Thank you!

**Pull request merged**